regions/core/compound.py

```diff
@@ -10,9 +10,17 @@ class CompoundPixelRegion(PixelRegion):
     Represents the logical combination of two regions in pixel coordinates.
     """
 
-    def __init__(self, region1, operator, region2):
+    def __init__(self, region1, operator, region2, meta=None, visual=None):
         self.region1 = region1
         self.region2 = region2
+        if meta is None:
+            self.meta = [region1.meta, region2.meta]
```

Comment:
`meta` and `visual` should be dictionaries, not a list.  We need to make some choice about which region's metadata will take priority; probably the first is best.

Reply:
Ok! We can probably take the first by default.!